docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -560,6 +580,9 @@ the 17 node.
 `edpm_tuned_profile` variable in the `OpenStackDataPlaneNodeSet` is configured
 to use the same profile as set on the (source) OSP 17 node.
 
+* if reboot check returned that reboot is required then make sure that the
```

Comment:
probably worth describing how to get the returned information from the ansibleee pod logs.